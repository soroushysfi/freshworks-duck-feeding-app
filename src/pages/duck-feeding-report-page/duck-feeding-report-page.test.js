import { shallow } from 'enzyme'
import DuckFeedingReportPage from './duck-feeding-report-page'


it('expect to match DuckFeedingReportPage snapshot', () => {
    expect(shallow(<DuckFeedingReportPage duckFeedingInfo = {[]} />)).toMatchSnapshot()

})