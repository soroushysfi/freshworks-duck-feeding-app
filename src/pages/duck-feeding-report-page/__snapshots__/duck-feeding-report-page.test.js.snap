// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to match DuckFeedingReportPage snapshot 1`] = `
<div
  className="box-format"
>
  <div
    className="lds-spinner"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
